templates/home/index.html

```diff
@@ -0,0 +1,215 @@
+        padding: 16px;
+      }
+      .tile__container {
+        margin: 28px 0 60px 0;
+      }
+      .tile__container a:not(:last-child) .tile {
+        margin-right: 24px;
+      }
+      .tile__icon {
+        height: 128px;
+        margin-bottom: 10px;
+      }
+      .tile__label {
+        display: block;
+        font-size: 16px;
+        font-weight: 600;
+        line-height: 170%;
+        transition: 200ms;
+      }
+      .tile:hover .tile__label {
+        color: #056dff;
+      }
+
+      @media (max-width: 1720px) {
+        h1,
+        h2 {
+          font-size: 36px;
+          line-height: 50px;
+        }
+        .container {
+          max-width: 1320px;
+        }
+
+        .content-container {
+          grid-template-columns: repeat(2, 1fr) 50px;
+          column-gap: 80px;
+        }
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="container">
+      <nav class="nav">
+        {% include "home/images/logo.svg" %}
+        <span> <a href="#">View documentation</a> for Saleor 2.10 </span>
+      </nav>
+      <div class="inner-container">
+        <div class="content-container">
+          <div class="splash__container">
+            {% include "home/images/splash.svg" %}
+          </div>
+          <main class="main">
+            <h1>Congratulations.</h1>
+            <h2>Your Saleor instance is ready to work</h2>
+
+            <p>
+              Now you can start setting up your ecommerce. Below are some useful
+              links you can use to start up:
+            </p>
+
+            <div class="tile__container">
+              {% if dashboard_url %}
+              <a href="{{dashboard_url}}" target="_blank">
+                <button class="btn tile">
+                  {% include "home/images/dashboard.svg" %}
+                  <span class="tile__label" href="#">Dashboard</span>
+                </button>
+              </a>
+              {% endif %} {% if storefront_url %}
+              <a href="{{storefront_url}}" target="_blank">
+                <button class="btn tile">
+                  {% include "home/images/storefront.svg" %}
+                  <span class="tile__label" href="#">Storefront</span>
+                </button>
+              </a>
+              {% endif %}
+
+              <a href={% url "api" %}  target="_blank">
+                <button class="btn tile">
+                  {% include "home/images/playground.svg" %}
+                  <span class="tile__label" href="#">GraphQL Playground</span>
+                </button>
+              </a>
+            </div>
+
+            {% if not storefront_url and not dashboard_url %}
+            <p>
+              It looks like you're only running the Saleor core. If you're
+              looking for an easy way to run all services locally, check out
+              <a
+                href="https://github.com/mirumee/saleor-platform"
+                target="_blank"
+                >saleor platform.</a
```

Comment:
I think it deserves it:
```suggestion
                >Saleor Platform.</a
```

Reply:
Copied straight out of figma 🤷‍♂️